code/__HELPERS/game.dm

```diff
@@ -486,7 +486,7 @@
   * * station_levels_only - Whether to only consider vents that are in a Z-level with a STATION_LEVEL trait
   * * z_level - The Z-level number to look for vents in. Defaults to all
   */
-/proc/get_valid_vent_spawns(unwelded_only = TRUE, exclude_mobs_nearby = FALSE, nearby_mobs_range = world.view, exclude_visible_by_mobs = FALSE, min_network_size = 50, station_levels_only = TRUE, z_level = 0)
+/proc/get_valid_vent_spawns(unwelded_only = TRUE, exclude_mobs_nearby = FALSE, nearby_mobs_range = world.view, exclude_visible_by_mobs = FALSE, min_network_size = 50, station_levels_only = TRUE, z_level = 0, only_exclude_mobs_with_mind = FALSE)
```

Comment:
Can this have fewer arguments please?
It is only ever called with either
`(exclude_mobs_nearby = TRUE, exclude_visible_by_mobs = TRUE, only_exclude_mobs_with_mind = TRUE)` or
`(min_network_size = 0, station_levels_only = FALSE, z_level = z)`

What do you think about compressing the former set to `unseen = TRUE` and latter to `min_network_size=0, z_level = z` and removing the rest? I'm fine with immediately declaring those arguments as *vars* (and leave the logic as is), i'm just scared by the unreadably large signature.

Reply:
there is also `(exclude_mobs_nearby = TRUE, exclude_visible_by_mobs = TRUE)` called without `only_exclude_mobs_with_mind `

This proc is a bit of a mess really, it's future-proofed for a purpose that never appeared